CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 ### Grafana Mimir
 
+* [FEATURE] Allow users to set type and internalTrafficPolicy setting on Kubernetes services
```

Comment:
Oh, I didn't notice this initially. The Helm chart has its own changelog in `operations/helm/charts/mimir-distributed/CHANGELOG.md`. I will fix it